main.js

```diff
@@ -44,143 +47,133 @@ form.addEventListener('submit', (e) => {
   }
 });
 
-document.addEventListener('DOMContentLoaded', () => {
-  const hamburger = document.querySelector('.hamburger');
-  const navMenu = document.querySelector('.nav-menu');
-  const close = document.querySelector('.close');
-  const open = document.querySelector('.fa-bars');
-  const navLink = document.querySelectorAll('.nav-link');
-  const modal = document.getElementById('card');
-
-  function toggleDisplay(element, val) {
-    element.style.display = val;
-  }
-  function mobileMenu() {
-    navMenu.classList.toggle('active');
-    toggleDisplay(open, 'none');
-    toggleDisplay(close, 'block');
-  }
-
-  function closeMenu() {
-    navMenu.classList.remove('active');
-    toggleDisplay(open, 'block');
-    toggleDisplay(close, 'none');
-  }
-
-  function switchIcons() {
-    toggleDisplay(open, 'block');
-    toggleDisplay(close, 'none');
-    closeMenu();
-  }
+// localStorage
+
+const addDataToLocalStorage = () => {
+  const myFormData = {
+    fullName: name.value,
+    email: mail.value,
+    message: msg.value,
+  };
+  localStorage.setItem('myFormData', JSON.stringify(myFormData));
+};
+form.addEventListener('change', addDataToLocalStorage);
```

Comment:
- [ ] Your code works but not as expected. Your inputs only saved to localStorage after removing focus. But they should save while typing. Simply replace 'change' with 'input'.